src/main/main.js

```diff
@@ -523,6 +524,16 @@ function initializeAfterAppReady() {
         });
     });
 
+    // Code for preventing user-agent checking. See: https://mattermost.atlassian.net/browse/MM-31626
```

Comment:
I'm not sure this is needed, I think it wasn't happening on browser-view

in that regard, we are currently changing the UA for BrowserViews as it wasn't being added by electron, but we might want to check that's still the case. if so, then the patch for gitlab might still be needed

Reply:
Yeah, looks like it works just fine so I'll remove this.